src/Sulu/Bundle/AdminBundle/Resources/js/services/Translator/Translator.js

```diff
@@ -0,0 +1,24 @@
+// @flow
+import type {TranslationMap} from './types';
+
+class Translator {
+    translations: ?TranslationMap;
+
+    set(translations: TranslationMap) {
+        this.translations = translations;
+    }
+
+    clear() {
+        this.translations = null;
+    }
+
+    translate(key: string) {
+        if (!this.translations || !(key in this.translations)) {
+            throw new Error('Translation for key "' + key + '" not found');
+        }
+
+        return this.translations[key];
+    }
+}
+
```

Comment:
add a `translate` function and export which can be used as shortcut for `translator. translate`